ui-test/package-lock.json

```diff
@@ -1303,6 +1304,20 @@
       "integrity": "sha512-E+XQCRwSbaaiChtv6k6Dwgc+bx+Bs6vuKJHHl5kox/BaKbhiXzqQOwK4cO22yElGp2OCmjwVhT3HmxgyPGnJfQ==",
       "dev": true
     },
+    "node_modules/call-bind-apply-helpers": {
```

Comment:
Could you rebase the code? It looks like I already updated this file yesterday
- https://github.com/apache/spark/pull/51689
Does it need to be changed again?